packages/client3/components/design-system/input/input.tsx

```diff
@@ -103,60 +109,109 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
         <label htmlFor={_id}>{label}</label>
         {labelDecoration ? labelDecoration : null}
       </div>
-      <div className="relative w-full">
-        <input
-          className={clsx(
-            "unfocused w-full rounded border",
-            "focus:border-primary focus:border-2",
-            "bg-transparent",
-            "placeholder:text-dark-80",
-            "text-dark-80",
 
-            isError ? "border-state-error" : "border-dark-80",
+      <div
+        className={clsx(
+          "relative mt-1 w-full rounded",
+          isMoneyInput ? "border" : null,
+          focus
+            ? [isError ? "ring-2 ring-state-error" : "ring ring-theme-primary"]
+            : "ring-theme-primary",
```

Comment:
You might want to merge main branch first, and fix those classes

Reply:
Yeap sure. Lots of branches to keep track of.